### PR TITLE
uffizzi 2.4.1

### PR DESCRIPTION
**Pull request by krehel**
Created with `brew bump-formula-pr`.

- [x] `resource` blocks have been checked for updates.
<details>
  <summary>release notes</summary>
  <pre>## What's Changed
* [388] updated upload artifacts workflow version by @moklidia in https://github.com/UffizziCloud/uffizzi_cli/pull/389

**Full Changelog**: https://github.com/UffizziCloud/uffizzi_cli/compare/v2.4.0...v2.4.1</pre>
</details>

### CODE REVIEW

**Comment by iMichka** [comment]
Some `gem` ressource versions got downgraded. How's that? `activesupport`, `tzinfo`, `addressable`, `public_suffix`

**Comment by chenrui333** [comment]
> Some `gem` ressource versions got downgraded. How's that? `activesupport`, `tzinfo`, `addressable`, `public_suffix`

I just did whatever the `gem list` shows

**Comment by chenrui333** [comment]
> > Some `gem` ressource versions got downgraded. How's that? `activesupport`, `tzinfo`, `addressable`, `public_suffix`
> 
> I just did whatever the `gem list` shows

that also matches with what in the gem.lock file, https://github.com/UffizziCloud/uffizzi_cli/blob/develop/Gemfile.lock#L19

File: Formula/u/uffizzi.rb [hunk 1]
```diff
@@ -91,13 +91,13 @@ class Uffizzi < Formula
...
+    url "https://rubygems.org/gems/public_suffix-5.0.3.gem"
+    sha256 "337d475da2bd2ea1de0446751cb972ad43243b4b00aa8cf91cb904fa593d3259"
   end
 
   resource "addressable" do
-    url "https://rubygems.org/gems/addressable-2.8.5.gem"
-    sha256 "63f0fbcde42edf116d6da98a9437f19dd1692152f1efa3fcc4741e443c772117"
+    url "https://rubygems.org/gems/addressable-2.8.4.gem"
+    sha256 "40a88af5285625b7fb14070e550e667d5b0cc91f748068701b4d897cacda4897"
   end
```
Comment (chenrui333) [inline Formula/u/uffizzi.rb:101]: not quite sure about why these got downgraded

File: Formula/u/uffizzi.rb [hunk 2]
```diff
@@ -161,13 +161,13 @@ class Uffizzi < Formula
...
 
   resource "activesupport" do
-    url "https://rubygems.org/gems/activesupport-7.1.2.gem"
-    sha256 "5bd81ef71524a14725ecb6c3b07bfc00ae0ccc77dd611190bd93bd9f92abd0d0"
+    url "https://rubygems.org/gems/activesupport-6.1.7.gem"
```
Comment (chenrui333) [inline Formula/u/uffizzi.rb:164]: https://github.com/UffizziCloud/uffizzi_cli/blob/develop/Gemfile.lock#L19

File: Formula/u/uffizzi.rb [hunk 3]
```diff
@@ -66,8 +66,8 @@ class Uffizzi < Formula
   end
 
   resource "thor" do
-    url "https://rubygems.org/gems/thor-1.3.0.gem"
-    sha256 "1adc7f9e5b3655a68c71393fee8bd0ad088d14ee8e83a0b73726f23cbb3ca7c3"
+    url "https://rubygems.org/gems/thor-1.2.1.gem"
```
Comment (chenrui333) [inline Formula/u/uffizzi.rb:69]: this seems a bit off, 

https://github.com/UffizziCloud/uffizzi_cli/blob/develop/Gemfile.lock#L105

**Comment by chenrui333** [comment]
I guess we can use whatever latest, refreshed the upstream gem lockfile, https://github.com/UffizziCloud/uffizzi_cli/pull/390

**Pull request closed without merging**